letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
-end
+  if integer >= 90
+    return "A"
```

Comment:
Awesome stuff, few quick things. 

1. In Ruby you do not need an explicit `return` it will return the last evaluated line of code in the block.

2. Sometimes conditionals can grow pretty quick and get ugly, what would be your next steps to refactor this logic?